inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sources/file/AbstractSource.java

```diff
@@ -309,6 +309,37 @@ private void initExtendHandler() {
 
     @Override
     public Message read() {
+        SourceData sourceData = readFromQueue();
+        while (sourceData != null) {
+            Message msg = createMessage(sourceData);
+            if (filterSourceData(msg)) {
```

Comment:
All message will be procced by `filterSourceData` with following logic
```java
if (extendedHandler != null) {
            return extendedHandler.filterMessage(msg);
        }
        return true;
```
But I didn't find any implementation for `ExtendedHandler#filterMessage`